/usr/include/c++/v1/__bsd_locale_fallbacks.h:123:17: error: use of undeclared identifier 'vasprintf'; did you mean 'vsprintf'?
zeek-archiver.cc:617:14: error: use of undeclared identifier 'strptime'

Index: auxil/zeek-archiver/zeek-archiver.cc
--- auxil/zeek-archiver/zeek-archiver.cc.orig
+++ auxil/zeek-archiver/zeek-archiver.cc
@@ -1,4 +1,4 @@
-#define _XOPEN_SOURCE
+// #define _XOPEN_SOURCE 700
 #include <time.h>
 
 #include <sys/time.h>
